src/Text/LLVM/PP.hs

```diff
@@ -971,7 +968,7 @@ ppDIImportedEntity = ppDIImportedEntity' ppLabel
 ppDILabel' :: Fmt i -> Fmt (DILabel' i)
 ppDILabel' pp ie = "!DILabel"
   <> parens (mcommas [ (("scope:"  <+>) . ppValMd' pp) <$> dilScope ie
-                     , pure ("name:" <+> text (dilName ie))
+                     , pure ("name:" <+> doubleQuotes (text (dilName ie)))
```

Comment:
I think this would be preferable:

```suggestion
                     , pure ("name:" <+> ppStringLiteral (dilName ie))
```

As `ppStringLiteral` performs escaping of certain characters.